controllers/topics/getLatestPost.js

```diff
@@ -0,0 +1,52 @@
+const {QueryTypes} = require('sequelize');
+const {sequelize, Topics} = require('../../databases/models');
+
+module.exports = async (req, res) => {
+  const {name} = req.query;
+  try {
+    let topics = await Topics.findOne({
+      where: {
+        name: name
+      }
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You're using `findOne` method to find a topic by its name. If the topic doesn't exist, you're returning a 400 status code which usually indicates a bad request. However, in this case, it would be more appropriate to return a 404 status code indicating that the resource was not found.



<!-- This is an auto-generated comment by OSS CodeRabbit -->